src/Membership/MembershipRequestHandler.php

```diff
@@ -69,4 +70,16 @@ private function generateMembershipActivationUrl(Adherent $adherent, AdherentAct
 
         return $this->urlGenerator->generate('app_membership_activate', $params, UrlGeneratorInterface::ABSOLUTE_URL);
     }
+
+    public function leftMembership(Adherent $adherent)
+    {
+        $message = AdherentLeftMembershipMessage::createFromAdherent($adherent);
+        $token = $this->manager->getRepository(AdherentActivationToken::class)->findOneBy(['adherentUuid' => $adherent->getUuid()->toString()]);
+
+        $this->manager->remove($token);
```

Comment:
Terminating membership requires to unfollow the committees. The precalculated "members count" for each committees must be recalculated to keep them up-to-date.